src/main/java/com/wolox/training/models/Book.java

```diff
@@ -84,54 +85,55 @@ public String getAuthor() {
     }
 
     public void setAuthor(String author) {
-        this.author = author;
+        this.author = Preconditions.checkNotNull(author, "Author must be not null");
```

Comment:
for more security, although the card does not indicate it, you can also add validation for empty strings, for example, `Preconditions.checkArgument(!Strings.isNullOrEmpty(author));`

Reply:
I agree. thanks.